possible misuse of unsafe.Pointer in gasyncresult.go

I tried to add these bindings to my app, and during compilation I got following error:

```
compilepkg: nogo: errors found by nogo during build-time code analysis:
/.../vendor/github.com/go-gst/go-glib/glib/__main__/vendor/github.com/go-gst/go-glib/glib/gasyncresult.go:86:55: possible misuse of unsafe.Pointer (unsafeptr)
```

I was able to silence this by adding `// nolint:unsafeptr` comment to this line. Please fix this or add this comment if this is a false positive.

--- a/glib/gasyncresult.go
+++ b/glib/gasyncresult.go
@@ -82,7 +82,7 @@ func (v *AsyncResult) GetSourceObject() *Object {
 }
 
 // IsTagged is a wrapper around g_async_result_is_tagged
-func (v *AsyncResult) IsTagged(sourceTag uintptr) bool {
+func (v *AsyncResult) IsTagged(sourceTag unsafe.Pointer) bool {
 	c := C.g_async_result_is_tagged(v.native(), C.gpointer(sourceTag))
 	return gobool(c)
 }

